person.rb

```diff
@@ -7,6 +7,7 @@ def initialize(age, name = 'unknown', parent_permmission: true)
     @name = name
     @age = age
     @parent_permmission = parent_permmission
+    @rentals = []
   end
 
```

Comment:
- [ ] **Required:** The Person class should also have a method to add rentals which return an instance of the Rental class.